pandas/tests/indexes/numeric/test_indexing.py

```diff
@@ -13,17 +13,28 @@ def index_large():
 
 
 class TestGetLoc:
-    def test_get_loc_float64(self):
+    @pytest.mark.parametrize("method", ["pad", "backfill", "nearest"])
+    def test_get_loc_float64_method(self, method):
         idx = Float64Index([0.0, 1.0, 2.0])
-        for method in [None, "pad", "backfill", "nearest"]:
-            assert idx.get_loc(1, method) == 1
-            if method is not None:
-                assert idx.get_loc(1, method, tolerance=0) == 1
 
-        for method, loc in [("pad", 1), ("backfill", 2), ("nearest", 1)]:
-            assert idx.get_loc(1.1, method) == loc
-            assert idx.get_loc(1.1, method, tolerance=0.9) == loc
+        assert idx.get_loc(1, method=method) == 1
+        assert idx.get_loc(1, method=method, tolerance=0) == 1
 
+    def test_get_loc_float64_method_none(self):
+        idx = Float64Index([0.0, 1.0, 2.0])
+        method = None
+        assert idx.get_loc(1, method=method) == 1
```

Comment:
to match how this looks in other files, lets remove "method_" from the test name, and not pass method kwarg

Reply:
@MomIsBestFriend can you address comments; this is generally a nice refactor